config/config.default.yaml

```
@@ -1012,6 +1012,12 @@ solving:
     cbc-default: {} # Used in CI
     glpk-default: {} # Used in CI
 
+  check_objective:
+    enable: false
+    expected_value: None
+    atol: 1000
+    rtol: 0.001
```

I would be more generous for the default (absolute value of 1000 may often be within solver tolerance).
```suggestion
    atol: 1000000
    rtol: 0.01
```